tools/ci_build/github/linux/docker/Dockerfile.manylinux2010_gpu

```diff
@@ -5,15 +5,10 @@ ARG PYTHON_VERSION=3.5
 ENV NVIDIA_VISIBLE_DEVICES all
 ENV NVIDIA_DRIVER_CAPABILITIES compute,utility
 
-SHELL ["/bin/bash", "-c"]
-
 ADD scripts /tmp/scripts
-RUN cd /tmp/scripts && \
-    /tmp/scripts/install_centos.sh && \
-    /tmp/scripts/install_deps.sh -d gpu -p $PYTHON_VERSION && \
-    yum remove -y devtoolset-\* && \
-    yum install -y devtoolset-7-binutils devtoolset-7-gcc devtoolset-7-gcc-c++ devtoolset-7-gcc-gfortran devtoolset-7-libquadmath-devel devtoolset-7-libstdc++-devel devtoolset-7-runtime && \
-    rm -rf /tmp/scripts
+RUN cd /tmp/scripts && /tmp/scripts/install_centos.sh && /tmp/scripts/install_deps.sh -p $PYTHON_VERSION && \
+yum remove -y devtoolset-\*  && yum install -y devtoolset-7-binutils devtoolset-7-gcc devtoolset-7-gcc-c++ devtoolset-7-gcc-gfortran devtoolset-7-libquadmath-devel devtoolset-7-libstdc++-devel devtoolset-7-runtime && \
```

Comment:
"yum remove -y devtoolset-\*  && yum install -y devtoolset-7-binutils devtoolset-7-gcc devtoolset-7-gcc-c++ devtoolset-7-gcc-gfortran devtoolset-7-libquadmath-devel devtoolset-7-libstdc++-devel devtoolset-7-runtime" can be removed I think. You may do it if you want.

Reply:
i tried not including it in the manylinux2014 image but got some error about CUDA 10.1 not supporting the gcc version. will leave it there